### PR TITLE
Refactored app skeleton

### PR DESCRIPTION
**Pull request by tmeasday**
See #6306 

 - Split files into the correct place, given the guide
 - Import html file
 - Removed test [it's not a demo]
 - Use ReactiveVar rather than Session
 - Added an info section to guide people further

### CODE REVIEW

**Comment by tmeasday** [comment]
@benjamn sending this to you because I undid some of the stuff we discussed :/ (based on discussions w/ others).

This PR takes some inspiration from #6347 (thank you @laosb), but doesn't go as far as styling the skeleton.

Files are named as they would be per the guide. We remove the test as people should do the tutorial if they want to learn about Meteor features.

**Comment by tmeasday** [comment]
A small question: I changed `"main"` in `package.json` to `server/main.js`. Should we add a `"client"` field?

File: tools/static-assets/skel/client/main.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { Template } from 'meteor/templating';
+import { ReactiveVar } from 'meteor/reactive-var';
+
+import './main.html';
+
+Template.hello.onCreated(function helloOnCreated() {
+  // counter starts at 0
+  this.counter = new ReactiveVar(0);
```
Comment (stubailo) [inline tools/static-assets/skel/client/main.js]: Why this instead of `ReactiveDict`?

File: tools/static-assets/skel/package.json [hunk 2]
```diff
@@ -2,10 +2,9 @@
   "name": "~name~",
   "private": true,
   "version": "0.0.1",
-  "main": "~name~.js",
+  "main": "server/main.js",
```
Comment (stubailo) [inline tools/static-assets/skel/package.json]: Should we just remove this field? IMO it's not meaningful to node.

File: tools/static-assets/skel/client/main.js (hunk 1 above)
Comment (tmeasday) [inline tools/static-assets/skel/client/main.js]: I guess the guide says use a dict huh. I'll change this.

File: tools/static-assets/skel/package.json (hunk 2 above)
Comment (tmeasday) [inline tools/static-assets/skel/package.json]: It could be premature. What do you think @benjamn?

**Pull request merged**